index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Week1</title>
+      <link rel="stylesheet" href="styles/style.css">
+   </head>
+   <body>
+      <header>
+         <div>
+            <div>
+               <a href=""> <img src="images/swiggylogo.png" alt = "Swiggy Logo" class = "swiggy-logo"> </a>
+               <a href="">Set location</a> 
+               <a href="">Click to Choose Location</a>
+            </div>
+            <ul>
+               <li><a href="">Help</a></li>
+               <li><a href="">Sign In</a></li>
+               <li><a href="">Cart</a></li>
+            </ul>
+         </div>
+      </header>
+      <section class = "nav-bar">
+         <nav>
+            <ul>
+               <li><a href="">Home</a> <span>/</span></li>
+               <li><a href="">Banglore</a> <span>/</span></li>
+               <li><a href="">Ashok Nagar</a> <span>/</span></li>
+               <li><a href="">Meghana Foods</a></li>
+            </ul>
+         </nav>
+      </section>
+      <section class="banner">
+         <img src="images/box1_image.jpeg" alt = "Food Image">
+         <h1>Punjabi Momos</h1>
+      </section>
+      <main class="row">
+         <div class="categories col-3">
+            <ul>
+               <li class = "category-recommended">
```

Comment:
Don't call this category-recommended as the top category may change to Favourites, Most frequently ordered or any other category in future. You can call this `highlight` or `category-highlight`